<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganizationTab</class>
 <widget class="QWidget" name="OrganizationTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="jobTitleLabel">
     <property name="text">
      <string>Job Title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="job_title_edit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="departmentLabel">
     <property name="text">
      <string>Department:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="department_edit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="companyLabel">
     <property name="text">
      <string>Company:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="company_edit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="managerLabel">
     <property name="text">
      <string>Manager:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ManagerWidget" name="manager_widget" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Reports:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTreeView" name="reports_view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ManagerWidget</class>
   <extends>QWidget</extends>
   <header>edits/manager_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
